Packs/Base/Scripts/DBotTrainTextClassifierV2/DBotTrainTextClassifierV2.py

```diff
@@ -351,6 +374,11 @@ def main():
     keyword_min_score = float(demisto.args()['keywordMinScore'])
     return_predictions_on_test_set = demisto.args().get('returnPredictionsOnTestSet', 'false') == 'true'
     original_text_fields = demisto.args().get('originalTextFields', '')
+    algorithm = demisto.args().get('trainingAlgorithm', AUTO_TRAINING_ALGO)
+    # FASTTEXT_TRAINING_ALGO and FROM_SCRATCH_TRAINING_ALGO are equivalent, replacement is done because ml_lib
```

Comment:
remove hidden code

Reply:
It's a comment, wanted to clarify why in the UI it's presented as "from_scratch" but in the script it's changed to "fasttext"